btsieve/config/bitcoin_basicauth.toml

```diff
@@ -0,0 +1,13 @@
+[bitcoin]
```

Comment:
We just discussed a testing strategy with @thomaseizinger involving multiple configuration files for `cnd`. I think it'd be good to do the same strategy for btsieve. Please chat with him.

Reply:
Conclusion after discussion with @thomaseizinger: should be done separate from this issue.